networkx/algorithms/simple_paths.py

```diff
@@ -324,6 +326,22 @@ def all_simple_edge_paths(G, source, target, cutoff=None):
         >>> len(paths)
         1
 
+    You can use the ``cutoff`` parameter to only generate paths that are
+    shorter than a certain length:
```

Comment:
I don't have an opinion w.r.t. either way of saying it being "more natural", but why doesn't this sentence also use something like
```suggestion
    You can use the `cutoff` parameter to only generate paths that contain <= `cutoff + 1` nodes:
```

Reply:
I worded it this way to match the corresponding existing sentence in the `all_simple_paths()` docstring:

https://github.com/networkx/networkx/blob/6eca9323b79cf093c6e7efb966c3bfbdbc031919/networkx/algorithms/simple_paths.py#L136-L141

I suppose one (small) benefit of this wording is that, if the behavior of the `cutoff` parameter is changed in the future, then this is one less thing to update. However, I don't have strong feelings about it either way -- if it is a blocking issue on this PR I am happy to adjust the wording here.

----

More generally, looking through the NetworkX codebase for the word `certain`, it looks like many functions use this word in a similar mildly ambiguous way:

https://github.com/networkx/networkx/blob/6eca9323b79cf093c6e7efb966c3bfbdbc031919/networkx/algorithms/planar_drawing.py#L11-L13

https://github.com/networkx/networkx/blob/6eca9323b79cf093c6e7efb966c3bfbdbc031919/networkx/generators/random_graphs.py#L1027-L1032

https://github.com/networkx/networkx/blob/6eca9323b79cf093c6e7efb966c3bfbdbc031919/networkx/algorithms/threshold.py#L946-L949

Hopefully updating all of these is out of scope of this PR :)